src/zenml/pipelines/base_pipeline.py

```diff
@@ -266,6 +272,7 @@ def configure(
         self: T,
         enable_cache: Optional[bool] = None,
         enable_artifact_metadata: Optional[bool] = None,
+        enable_artifact_visualization: Optional[bool] = None,
```

Comment:
I didn't see this new configuration attribute documented anywhere. It would be a shame if it just goes unnoticed. Maybe you could add a few lines in the section on `Runtime Settings` ?

Reply:
I added some info as an info box in the materializer docs. Also adjusted the runtime settings page to include this in the example config 👍